src/components/pages/Books.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import Book from './Book';
+import Form from './Form';
+
+const bookList = [
+  {
+    id: 'book1',
+    title: 'Things Fall Apart',
+    author: 'Chinua Achebe'
+  },
+  {
+    id: 'book2',
+    title: 'Life is Good',
+    author: "Adelakun Abdulwasi'"
+  }
+];
+
+const Books = () => (
+  <>
+    <div className="container">
+      <h1 className="booklist-header">List of Books</h1>
+
+      {bookList.map((book) => (
+        <div className="booklist" key={book.id}>
+          <Book book={book} />
+        </div>
+      ))}
+
+      <NavLink className="btn link">Remove Book</NavLink>
```

Comment:
you only have one remove button, how will this button determine which book to remove?.. although the button's functionality isn't done yet, it will be a better practice to move the button into the `<book />` component. This way each book has it's button

Reply:
I have made the correction. Thanks